index.js

```diff
@@ -22,4 +22,38 @@ addTaskButton.addEventListener("click", () => {
   });
 
   console.log(tasks);
+
+  taskTable = document.getElementById("task-table");
+  const TaskDetails = document.createElement("tr");
+  TaskDetails.className = "table-box-row";
+
+  for (let taskArrayIndex in tasks) {
+    for (
+      let taskObjectIndex = 0;
+      taskObjectIndex < Object.values(tasks[taskArrayIndex]).length;
+      taskObjectIndex++
+    ) {
+
+      TaskDetails.innerHTML =
+        `<td class="table-box-color-tag-job">` +
+        "" +
+        `</td>` +
+        `<td class="table-box-task" id="show-task-title">` +
+        Object.values(tasks[taskArrayIndex])[1] +
+        `</td>` +
+        `<td class="table-box-time" id="show-task-time">` +
+        Object.values(tasks[taskArrayIndex])[3].toString().match(/\d\d:\d\d:\d\d/) +
```

Comment:
define and assign variables before string concatenation. Like:
```js
const taskCreatedAt =  // whatever logic you have
```

Reply:
greaaat idea,I did it ^^